MantidQt/CustomInterfaces/src/Muon/MuonAnalysis.cpp

```diff
@@ -1528,7 +1528,14 @@ void MuonAnalysis::updateFrontAndCombo(bool updateIndexAndPlot) {
   if (currentI >= m_uiForm.frontGroupGroupPairComboBox->count()) {
     currentI = 0;
   }
-  m_uiForm.fitBrowser->setAvailableGroups(groupsAndPairs);
+  auto tmp = m_groupingHelper.parseGroupingTable().groupNames;
+  auto names = m_groupingHelper.parseGroupingTable().pairNames;
+  names.insert(std::end(names), std::begin(tmp), std::end(tmp));
```

Comment:
Since we don't reuse `tmp` can we move the elements instead of copying them: [link](https://stackoverflow.com/a/10720242)